src/ui/public/render_directive/render_directive.js

```diff
@@ -6,7 +6,8 @@ module.directive('renderDirective', function () {
   return {
     restrict: 'E',
     scope: {
-      'definition': '='
+      'definition': '=',
+      'scope': '=?'
```

Comment:
:pensive: not sure how we can make this better but I hope we can come up with something. Referring to `$scope.scope` internally is gross.